src/__test__/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Header from '@components/Header';
+import { render } from '@testing-library/react';
+import * as SidebarContext from '@store/sidebarContext';
+
+describe('Header component', () => {
+  it('should be able to render the Header component', () => {
+    const { getByText, getByPlaceholderText, getByTestId } = render(
+      <Header isComparisonModeOn={false} comparisonType="table" setComparisonType={() => {}} />,
+    );
+
+    const regionSearchInput = getByPlaceholderText('Pesquise por regiões');
+    const layerRoute = getByText('Brasil');
+    const drawerButton = getByTestId('MenuIcon');
+
+    expect(regionSearchInput).toBeTruthy();
+    expect(layerRoute).toBeTruthy();
+    expect(drawerButton).toBeTruthy();
```

Comment:
Since we are using `testing-library`, we can use the `screen` that verify **behind the scene** and back warning if have a error , without use the `expect`

```suggestion
    screen.getByPlaceholderText('Pesquise por regiões');
    screen.getByText('Brasil');
    screen.getByTestId('MenuIcon');
```